### PR TITLE
JBIDE-26439 - Update WF from 11 to 14

### PR DESCRIPTION
**Pull request by zcervink**
Updated WF from 11 to 14. Test report will be added soon. Please review @vprusa.

### CODE REVIEW

File: tests/org.jboss.tools.examples.ui.bot.test/README.md [hunk 1]
```diff
@@ -3,18 +3,18 @@ Executing tests:
...
--PWildFly11
+-PWildFly14
 -DquickstartsURLWildFly=${quickstartsURLWildFly}		-	URL for WildFly Quickstarts
 -DexamplesFolderWildFly=${examplesFolderWildFly}		-	examples folder of WildFly Quickstarts
 
 G.e.
 
 ```
-mvn clean verify -DexamplesFolderWildFly=wildfly-11.0.0.Final-quickstarts -DquickstartsURLWildFly=https://github.com/wildfly/quickstart/releases/download/11.0.0.Final/wildfly-11.0.0.Final-quickstarts.zip -PWildFly -DspecificQuickstarts=helloworld-jms
+mvn clean verify -DexamplesFolderWildFly=wildfly-14.0.0.Final-quickstarts -DquickstartsURLWildFly=https://github.com/wildfly/quickstart/releases/download/14.0.0.Final/wildfly-14.0.0.Final-quickstarts.zip -PWildFly -DspecificQuickstarts=helloworld-jms
```
Comment (jkopriva) [inline tests/org.jboss.tools.examples.ui.bot.test/README.md]: in pom is profile WildFly14, can you please also change this line?

File: tests/org.jboss.tools.examples.ui.bot.test/pom.xml [hunk 2]
```diff
@@ -16,16 +16,16 @@
 		<!-- <systemProperties>${integrationTestsSystemProperties} -->
 		<!-- -DimportTestDefinition=${importTestDefinition}</systemProperties> -->
 		<systemProperties>${integrationTestsSystemProperties} -DexamplesLocation=${project.build.directory}/${examplesFolder} -Dmaven.settings.path=${maven.settings.path} -Drd.config=${reddeer.config} -DspecificQuickstarts=${specificQuickstarts} -DdeployOnServer=${deployOnServer}</systemProperties>
-		<quickstartsURLWildFly>https://github.com/wildfly/quickstart/archive/11.0.0.Final.zip</quickstartsURLWildFly>
+		<quickstartsURLWildFly>https://github.com/wildfly/quickstart/archive/14.0.0.Final.zip</quickstartsURLWildFly>
```
Comment (jkopriva) [inline tests/org.jboss.tools.examples.ui.bot.test/pom.xml]: There is also newer version released 14.0.1, I think we can use this one?

**Comment by jkopriva** [comment]
@zcervink Can you please also share a link to at least yellow build?

File: tests/org.jboss.tools.examples.ui.bot.test/README.md [hunk 3]
```diff
@@ -3,18 +3,18 @@ Executing tests:
 mvn clean install -P{profile} -Dswtbot.test.skip=true -Dusage_reporting_enable=false -DdeployOnServer=true
 
 Where
-	${profile} select maven profile: WildFly11, EAP7, JavaEE7, SmokeTestsEAP7, SmokeTestsWildFly10, SmokeTestsJavaEE7 (WildFly10 profile is selected, if you don't specify profile)
+	${profile} select maven profile: WildFly14, EAP7, JavaEE7, SmokeTestsEAP7, SmokeTestsWildFly10, SmokeTestsJavaEE7 (WildFly10 profile is selected, if you don't specify profile)
```
Comment (jkopriva) [inline tests/org.jboss.tools.examples.ui.bot.test/README.md]: please update also **WildFly10 profile is selected**

File: tests/org.jboss.tools.examples.ui.bot.test/resources/servers/wildfly-14.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+	"org.jboss.ide.eclipse.as.reddeer.server.requirement.ServerRequirement.JBossServer": [
+		{
+			"runtime": "${jbosstools.test.wildfly14.home}",
+			"version": "14",
+			"family": "WILDFLY"
+		}
+	]
+}
```
Comment (jkopriva) [inline tests/org.jboss.tools.examples.ui.bot.test/resources/servers/wildfly-14.json]: I think old json file could be removed, right?

Comment (zcervink) [reply tests/org.jboss.tools.examples.ui.bot.test/resources/servers/wildfly-14.json]: old wildfly-11.json removed.

**Comment by zcervink** [comment]
Link to yellow jenkins build:
https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/examples.wildfly.it.weekly/430/

**Comment by zcervink** [comment]
> @zcervink Can you please also share a link to at least yellow build?

Link to yellow jenkins build:
https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/examples.wildfly.it.weekly/430/

File: tests/org.jboss.tools.examples.ui.bot.test/resources/servers/wildfly-14.json (hunk 4 above)
Comment (zcervink) [reply tests/org.jboss.tools.examples.ui.bot.test/resources/servers/wildfly-14.json]: commit is also signed and pushed to github.

File: tests/org.jboss.tools.examples.ui.bot.test/README.md (hunk 1 above)
Comment (zcervink) [reply tests/org.jboss.tools.examples.ui.bot.test/README.md]: Fixed in commit "JBIDE-26439 - updates README.md in org.jboss.tools.examples.ui.bot.test - changed maven code from -PWildFly to -PWildFly14 to match the POM profile."

**Comment by jkopriva** [comment]
LGTM, merging.
Thanks @zcervink :)

**Pull request merged**